tests/apps/mip/test_config_mip.py

```diff
@@ -0,0 +1,151 @@
+import pytest
+from cg.exc import ConfigError
+from cg.apps.mip.confighandler import ConfigHandler
+
+def test_validate_config():
+    """Test to validate a correct config"""
+    # GIVEN a correct config
+    sample = dict(
```

Comment:
Looks like this could be "mip_pedigree_config" fixture toegther with the config = dict below, which you can import and modify according to what you want to test. Then there is one model for that can be reused